buildtest/cli/config.py

```diff
@@ -118,6 +121,11 @@ def validate_config(configuration):
     console.print(f"{configuration.file} is valid")
 
 
+def view_path(configuration):
+    """Display the path to configuration file regardless if file is valid"""
+    console.print(f"Config path: {configuration.file}")
```

Comment:
I am thinking we just print the file name without any text. Could be useful if one wants to do something like
```
ls -l $(buildtest config path)
cat $(buildtest config path)
cat $(buildtest config path) | grep
```